service/src/main/java/com/epam/rft/atsy/service/impl/LogicallyDeletableAbstractServiceImpl.java

```diff
@@ -12,16 +12,16 @@
 
 import java.util.List;
 
-public class LogicallyDeletableServiceImpl<D extends LogicallyDeletableDTO, E extends LogicallyDeletableEntity>
+public abstract class LogicallyDeletableAbstractServiceImpl<D extends LogicallyDeletableDTO, E extends LogicallyDeletableEntity>
```

Comment:
I prefer AbstractLogicallyDeletableService name instead of LogicallyDeletableAbstractServiceImpl.